website/client/src/components/chat/chatCard.vue

```diff
@@ -278,6 +280,13 @@ export default {
       return date.toString();
     },
   },
+  beforeMount () {
```

Comment:
thanks @mario130 , we discussed this internally and for now we'd like to limit the changes to the `dir` attribute, without changing the alignment, it's not a perfect solution but it's a start